llvm/test/tools/llvm-gsymutil/X86/Inputs/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+#include <unistd.h>
+
+void foo() {
+    std::cout << "This is foo" << std::endl;
+}
+
+int main() {
+    std::cout << "hello world" << std::endl;
+    foo();
+    std::cout << "after foo" << std::endl;
+    return 0;
+}
```

Comment:
Do you even need this file? In your test you are using a `cat` mechanism, so this isn't needed? Either switch your tests to use this file instead of `cat`, or remove the `cat` and use this file.

Reply:
I don't think cat is actually being used. Played with few other exisiting tests that does the same (which still checked in the binaries). I will double check again to see if cat works :D